client/src/sagas/alert.js

```diff
@@ -0,0 +1,24 @@
+import { delay, put, takeEvery } from 'redux-saga/effects';
+import { uuid } from 'uuidv4';
+
+import { alert } from './types';
+
+export function* handleAddAlert({ payload }) {
+  const id = uuid();
```

Comment:
Is this the recommended way to identify alerts?

Reply:
It was an easy way to get a unique ID. Open to suggestions if you don't think this is appropriate.

Reply:
🤷  i have no idea